### PR TITLE
[openthread] use latest OT commit

### PR DESCRIPTION
**Pull request by jwhui**
To fix the `./script/bootstrap` error.

Depends on https://github.com/openthread/openthread/pull/5668

### CODE REVIEW

**Comment by jwhui** [comment]
@simonlingoogle , can you help look into the Test / Unittests failures?

**Comment by simonlingoogle** [comment]
> @simonlingoogle , can you help look into the Test / Unittests failures?

I think you need to set env MAX_NETWORK_SIZE=999 for tests since we have merged https://github.com/openthread/openthread/pull/5565.

File: .github/workflows/stress.yml [hunk 1]
```diff
@@ -29,7 +29,7 @@ name: Stress
 on: [push, pull_request]
 
 env:
-  OT_COMMIT: "e48a1ff9807f66d448a9e76ad64038dc6c67adce" # Oct 17, 2020
+  OT_COMMIT: "7ba1f8d81d4ab2c44bd5a879269fadd56581f6bb" # Oct 19, 2020
```
Comment (simonlingoogle) [inline .github/workflows/stress.yml:32]:
```suggestion
  OT_COMMIT: "7ba1f8d81d4ab2c44bd5a879269fadd56581f6bb" # Oct 19, 2020
  MAX_NETWORK_SIZE: 999
```

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -29,7 +29,7 @@ name: Test
 on: [push, pull_request]
 
 env:
-  OT_COMMIT: "e48a1ff9807f66d448a9e76ad64038dc6c67adce" # Oct 17, 2020
+  OT_COMMIT: "7ba1f8d81d4ab2c44bd5a879269fadd56581f6bb" # Oct 19, 2020
```
Comment (simonlingoogle) [inline .github/workflows/test.yml:32]:
```suggestion
  OT_COMMIT: "7ba1f8d81d4ab2c44bd5a879269fadd56581f6bb" # Oct 19, 2020
  MAX_NETWORK_SIZE: 999
```

**Review by wgtdkp — APPROVED** [review]
LGTM :+1:

**Review by simonlingoogle — APPROVED** [review]
LGTM 👍

**Pull request merged**